packages/xdg_directories/pubspec.yaml

```diff
@@ -1,15 +1,15 @@
 name: xdg_directories
 description: A Dart package for reading XDG directory configuration information on Linux.
-version: 0.1.1
+version: 0.1.2
 homepage: https://github.com/flutter/packages/tree/master/packages/xdg_directories
 
 environment:
   sdk: ">=2.3.0 <3.0.0"
 
 dependencies:
-  meta: ^1.2.2
-  path: ^1.6.4
-  process: ^3.0.12
+  meta: ">=1.2.2 <2.0.0"
```

Comment:
Maybe <3.0.0 so we don't need to do this again when meta moves to 2.0.0. A 2.0.0-nullsafe version should be compatible with 2.x.x.

Reply:
Meta's nullsafe version is `1.3.0-nullsafety.2` at the moment.  I assume that after the pre-release is released, it'll be `1.3.0`, right? Or will they all get their major version bumped again?